frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/types.ts

```diff
@@ -14,7 +14,7 @@ export enum DiskType {
   Partition = 'Partition',
 }
 
-export enum DiskMechanicalProperty {
+export enum DiskMechanicalProperties {
```

Comment:
```suggestion
```
Since it can only have one property of either SSD or HDD I find it perfect as it is (in singular)